migrations/sql/V2022.02.01.12.46__update_project_summary_status_codes.sql

```diff
@@ -0,0 +1,31 @@
+ALTER TABLE project_summary_status_code add column alias_description character varying(100);
```

Comment:
why 'alias' for the descriptor on this description?

Reply:
Core and MineSpacfe display the labels differently!